specification/machinelearningcompute/resource-manager/Microsoft.MachineLearningCompute/2017-08-01-preview/machineLearningCompute.json

```diff
@@ -658,8 +658,7 @@
       "type": "object",
       "description": "Information about the container service backing the cluster",
       "required": [
-        "orchestratorType",
-        "orchestratorProperties"
```

Comment:
After conferring with @fearthecowboy, we agree that this should cause a breaking change because it is strictly placing what would have previously been the last required field and turns it into the first optional field. There aren't any flattening concerns, so this sort of lucks out and isn't breaking.